.github/workflows/test.yml

```diff
@@ -82,46 +82,6 @@ jobs:
           du -hd1 /__w /github || true
 
   Linux-self-hosted:
```

Comment:
Considering that we do not use GitHub runners in the Linux-self-hosted step, df and du are not very relevant for our runners. Additionally, the path /__w /github is also irrelevant. We can delete this steps.